### PR TITLE
Default Location Service & Dockerizing gitshots

### PR DESCRIPTION
**Pull request by adaofeliz**

### CODE REVIEW

File: hooks/post-commit.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 
 def run_command(command):
-    return subprocess.check_output(command.split(), shell=True).rstrip()
+    return subprocess.check_output(command.split()).rstrip()
```
Comment (ranman) [inline hooks/post-commit.py]: I don't remember why this was here (I think it was to do with aliasing and x-platform) but why did you take it out -- does it break something?

Comment (adaofeliz) [inline hooks/post-commit.py]: Sorry for not comment it. When I run this line with shell=True I get the following error:

Traceback (most recent call last):
  File "/Users/adaofeliz/bin/post-commit.py", line 11, in <module>
    tld = run_command('git rev-parse --show-toplevel')
  File "/Users/adaofeliz/bin/post-commit.py", line 8, in run_command
    return subprocess.check_output(command.split(), shell=True).rstrip()
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/subprocess.py", line 573, in check_output
    raise CalledProcessError(retcode, cmd, output=output)
subprocess.CalledProcessError: Command '['git', 'rev-parse', '--show-toplevel']' returned non-zero exit status 1

By removing it, this errors does not occur.

Comment (adaofeliz) [inline hooks/post-commit.py]: + Info: https://docs.python.org/2/library/subprocess.html#frequently-used-arguments

Comment (ranman) [inline hooks/post-commit.py]: no worries -- I have no idea why I did this originally (I first wrote this code in 2010)... so lets take it out and see what happens. This isn't rocket science.

**Pull request merged**